tmux/demos/medical/Kitware-R18.yml

```diff
@@ -105,6 +105,7 @@ windows:
       -p det_topic:=ObjectDetections2d
       -p object_net_checkpoint:=${MODEL_DIR}/object_detector/r18_det.pt
       -p hand_net_checkpoint:=${MODEL_DIR}/object_detector/hands_model.pt
+      -p cuda_device_id:=1
```

Comment:
Let's keep the model device changes local because that is going to be machine-dependent.